unittests/adt/klee/cv/resize.cpp

```diff
@@ -0,0 +1,63 @@
+//===- unittests/adt/klee/cv/resize.cpp -----------------------------------===//
+//*                _          *
+//*  _ __ ___  ___(_)_______  *
+//* | '__/ _ \/ __| |_  / _ \ *
+//* | | |  __/\__ \ |/ /  __/ *
+//* |_|  \___||___/_/___\___| *
+//*                           *
+//===----------------------------------------------------------------------===//
+//
+// Part of the pstore project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://github.com/SNSystems/pstore/blob/master/LICENSE.txt for license
+// information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+/// \file resize.cpp
+/// \brief KLEE test for chunked_vector<>::resize().
+///
+/// This test makes two calls to resize() both using symbolic arguments. The idea behind this is
+/// that we start with an empty container, resize it and then resize it again. The second of these
+/// resize operations may be a no-op or will enlarge or shrink the container from the starting point
+/// produced by the first resize. This should enable the test to cover all of the possible start and
+/// end conditions.
+
+#include <algorithm>
+#include <cassert>
+#include <iterator>
+
+#include <klee/klee.h>
+
+#include "pstore/adt/chunked_vector.hpp"
+
+namespace {
+
+    constexpr auto elements_per_chunk = 3U;
+    constexpr auto max_size = elements_per_chunk * 3U;
+
+    void check (pstore::chunked_vector<int, elements_per_chunk> const & cv,
+                std::size_t const size) {
+        assert (cv.size () == size);
+        assert (std::distance (std::begin (cv), std::end (cv) == size);
```

Comment:
Hmm. This doesn’t look right to me.

Reply:
Yes, failed in [this run](https://github.com/SNSystems/pstore/runs/2727833554?check_suite_focus=true).

Reply:
Fixed by commit 15e41a01b0f556037b9e6fe5217cef99a41613ae.